src/main/java/org/mandas/docker/client/DefaultLogStream.java

```diff
@@ -23,16 +23,14 @@
 import static com.google.common.base.Charsets.UTF_8;
 import static com.google.common.base.Throwables.throwIfUnchecked;
 
-import com.google.common.annotations.VisibleForTesting;
-import com.google.common.base.Throwables;
-import com.google.common.collect.AbstractIterator;
-import com.google.common.io.Closer;
-
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.nio.ByteBuffer;
 
+import com.google.common.collect.AbstractIterator;
+import com.google.common.io.Closer;
```

Comment:
Is this still a work in progress?  If not, were these missed?

Reply:
done